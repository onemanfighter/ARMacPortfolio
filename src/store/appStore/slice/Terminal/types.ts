export type TerminalState = {};

export interface TerminalAppAction {}

export type TerminalStateSlice = TerminalState & TerminalAppAction;
