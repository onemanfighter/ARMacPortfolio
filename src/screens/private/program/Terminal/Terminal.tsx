import { TerminalProps } from './type';

const Terminal = (props: TerminalProps) => {
  return (
    <div>
      <h1>Terminal</h1>
    </div>
  );
};

export default Terminal;
