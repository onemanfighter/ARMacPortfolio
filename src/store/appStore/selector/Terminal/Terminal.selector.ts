import { AppStoreState } from '../../appStore';

const terminalSelector = (state: AppStoreState) => ({});

export { terminalSelector };
