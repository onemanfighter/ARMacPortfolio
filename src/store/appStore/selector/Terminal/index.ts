export { terminalSelector } from './Terminal.selector';
