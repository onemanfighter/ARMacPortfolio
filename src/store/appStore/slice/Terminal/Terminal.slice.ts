import { AppStoreSlice } from '../../appStore';
import { TerminalState, TerminalStateSlice } from './types';

const defaultTerminalState: TerminalState = {};

const createTerminalSlice: AppStoreSlice<TerminalStateSlice> = (set) => ({
  ...defaultTerminalState,
});

export default createTerminalSlice;
