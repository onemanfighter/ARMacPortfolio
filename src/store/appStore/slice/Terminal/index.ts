export { default as createTerminalSlice } from './Terminal.slice';
export type { TerminalStateSlice, TerminalState } from './types';
