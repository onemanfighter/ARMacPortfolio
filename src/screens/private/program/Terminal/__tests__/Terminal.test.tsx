import { render } from '@testing-library/react';
import Terminal from '../Terminal';

describe('Terminal', () => {
  it('should render correctly to match snapshot', () => {
    const { container } = render(<Terminal />);

    expect(container).toMatchSnapshot();
  });
});
