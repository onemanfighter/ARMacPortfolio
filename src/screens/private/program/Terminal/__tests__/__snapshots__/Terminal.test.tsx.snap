// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Terminal should render correctly to match snapshot 1`] = `
<div>
  <div>
    <h1>
      Terminal
    </h1>
  </div>
</div>
`;
